components/Inventory/basic_usage.rst

```diff
@@ -0,0 +1,272 @@
+
+    //Let assume that we got 9 inventory units with state 'backordered'.
+    //This method will find all inventory units for that specific stockable with 'backordered' state.
+    $backordersHandler->fillBackorders($stockable);
+    // Now 6 of them will have 'sold' state.
+
+    $stockable->getOnHand(); //Output will be 0.
+
+.. note::
+    For more detailed information go to `Sylius API BackordersHandler`_.
+
+.. _Sylius API InventoryUnitFactory: http://api.sylius.org/Sylius/Component/Inventory/Operator/BackordersHandler.html
+
+
+AvailabilityChecker
+-------------------
+
+The **AvailabilityChecker** checks availability for given stockable object.
+To charactrize an object which is an **AvailabilityChecker**, it needs to implement the :ref:`component_inventory_checker_availability_checker_interface`_.
+Second parameter in ``->isStockSufficient()`` method indicates that customer want a merchandise in specific quantity.
+
+.. code:: php
+
+    <?php
+
+    use Sylius\Component\Inventory\Checker\AvailabilityChecker;
+
+    $stockable; //Stockable model.
+    $stockable->isAvailableOnDemand(); //Output will be false.
+    $stockable->getOnHand(); //Output will be 5
+    $stockable->getOnHold(); //Output will be 4
+
+    $availabilityChecker = new AvailabilityChecker(false); //backorders = false;
+    $availabilityChecker->isStockAvailable($stockable); //Output will be true.
+    $availabilityChecker->isStockSufficient($stockable, 5); //Output will be false.
+
+Backorders
+~~~~~~~~~~
+
+The backorder property generally indicates that customer demand for a product or service
+exceeds a stockable's capacity to supply it.
+
+.. code:: php
+
+    <?php
+
+    use Sylius\Component\Inventory\Checker\AvailabilityChecker;
+
+    $stockable; //Stockable model.
+
+    $availabilityChecker = new AvailabilityChecker(true); //backorders = true;
+    $availabilityChecker->isStockAvailable($stockable); //Output will be true.
+    $availabilityChecker->isStockSufficient($stockable, 5); //Output will be true.
+
+Available On Demand
+~~~~~~~~~~~~~~~~~~~
+
+.. code:: php
+
+    <?php
+
+    use Sylius\Component\Inventory\Checker\AvailabilityChecker;
+
+    $stockable; //Stockable model.
+    $stockable->isAvailableOnDemand(); //Output will be true.
+
+    $availabilityChecker = new AvailabilityChecker(false); //backorders = false;
+    $availabilityChecker->isStockAvailable($stockable); //Output will be true.
+    $availabilityChecker->isStockSufficient($stockable, 5); //Output will be true.
+
+.. hint::
+    In above cases value of ``->getOnHand()`` and ``->getOnHold()`` will be irrelevant.
+
+.. note::
+    For more detailed information go to `Sylius API AvailabilityChecker`_.
+
+.. _Sylius API AvailabilityChecker: http://api.sylius.org/Sylius/Component/Inventory/Checker/AvailabilityChecker.html
+
+InventoryUnitFactory
+--------------------
+
+The **InventoryUnitFactory** basically is simple factory. It creates collection of new inventory units.
```

Comment:
Thank you Mr. Obvious! :)

`The **InventoryUnitFactory** creates a collection of new inventory units.`